config/config.txt

```diff
@@ -176,10 +176,10 @@ GUEST_BAN
 # WIKIURL http://www.tgstation13.org/wiki
 
 ## Rules address
-# RULESURL http://www.tgstation13.org/wiki/Rules
+RULESURL https://discord.gg/yjCSCzzezp
```

Comment:
Putting your rules on discord seems like a bad idea and dick move for anybody who doesn't want to use discord, but config is your territory.

Reply:
Yes, that needs to be changed in the future; we're moving the rules to the wiki anyway.